File: db4oj.tests/src/com/db4o/db4ounit/jre11/types/AllTests.java
/* Copyright (C) 2004 - 2006 db4objects Inc. http://www.db4o.com */

package com.db4o.db4ounit.jre11.types;

import db4ounit.extensions.*;

public class AllTests extends Db4oTestSuite {

	public static void main(String[] args) {
		new AllTests().runSolo();
    }

	protected Class[] testCases() {
		return new Class[] {
				BigDecimalTestCase.class,
				BigIntegerTestCase.class,
				com.db4o.db4ounit.jre11.types.arrays.AllTests.class,
				com.db4o.db4ounit.jre11.types.collections.AllTests.class,
		};
	}
}

File: db4o.net/Db4objects.Db4o.Tests/Db4objects.Db4o.Tests/Common/Diagnostics/AllTests.cs
/* Copyright (C) 2004 - 2009  Versant Inc.  http://www.db4o.com */

using System;
using Db4oUnit.Extensions;
using Db4objects.Db4o.Tests.Common.Diagnostics;

namespace Db4objects.Db4o.Tests.Common.Diagnostics
{
	public class AllTests : Db4oTestSuite
	{
		public static void Main(string[] args)
		{
			new Db4objects.Db4o.Tests.Common.Diagnostics.AllTests().RunAll();
		}

		protected override Type[] TestCases()
		{
			return new Type[] { typeof(ClassHasNoFieldsTestCase), typeof(DescendIntoTranslatorTestCase
				), typeof(DiagnosticsTestCase), typeof(IndexFieldDiagnosticTestCase) };
		}
	}
}
